sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/changestreams/it/BigtableChangeStreamIT.java

```diff
@@ -145,13 +145,16 @@ public static void afterClass() {
 
   @Test
   public void testReadBigtableChangeStream() throws InterruptedException {
+    // Sleep to ensure clock discrepancy between client and server don't affect the start time.
+    Thread.sleep(500);
```

Comment:
Could we just relax the start time and stop time, and create a transient table so that if other test runs concurrently it won't interfere each other?